ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.ratis.server.DataStreamServerRpc;
 import org.apache.ratis.server.RaftServer;
 import org.apache.ratis.server.RaftServerConfigKeys;
+import org.apache.ratis.thirdparty.com.google.common.util.concurrent.ThreadFactoryBuilder;
```

Comment:
Use ConcurrentUtils.newThreadFactory(..) instead.  We should avoid external dependency.